docs/source/projects.rst

```diff
@@ -109,15 +98,14 @@ For example, to store data associated with particular temperature or pressure of
 .. tip::
 
     You only need to call the :meth:`Job.init` function the first time that you are accessing a job.
-    Furthermore, the :meth:`Job.init` function returns itself, so you can abbreviate like this:
+    Furthermore, the :meth:`Job.init` function returns the job itself, so you can abbreviate like this:
 
     .. code-block:: python
 
         job = project.open_job({"temperature": 20, "pressure": 1.0}).init()
 
-The job *state point* represents a **unique address** of your data within one project.
-There can never be two jobs that share the same *state point* within the same project.
-Any other kind of data and metadata that describe your job, but do not represent a unique address should be stored within the :attr:`Job.doc`, which has the exact same interface like the :attr:`Job.sp`, but does not represent a unique address of the job.
+The uniqueness of a state point should be familiar to anyone who has used a relational database: the state point parameters constitute the primary key of the data space.
```

Comment:
Rewrite to be inclusive of people who don't know about relational databases. Something like:
```suggestion
The uniqueness of a state point is like that in a relational database where the state point parameters are the primary key of the data space.
```

Reply:
Small rewrite to clarify the analogy:
```suggestion
The uniqueness of a state point in a signac data space is analogous to the uniqueness of a primary key in a relational database.
```